romancal/regtest/test_match_skycell.py

```diff
@@ -19,12 +19,8 @@
 ]
 
 EXPECTED_ASN_FILENAMES = [
-    "r00001_p_v0000101001001001001_r274dp63x31y80_f158_asn.json",
-    "r00001_p_v0000101001001001001_r274dp63x31y81_f158_asn.json",
-    "r00001_p_v0000101001001001001_r274dp63x31y82_f158_asn.json",
-    "r00001_p_v0000101001001001001_r274dp63x32y80_f158_asn.json",
-    "r00001_p_v0000101001001001001_r274dp63x32y81_f158_asn.json",
-    "r00001_p_v0000101001001001001_r274dp63x32y82_f158_asn.json",
+    "r00001_p_v0000101001001001001_270p65x49y70_f158_asn.json",
+    "r00001_p_v0000101001001001001_270p65x51y70_f158_asn.json",
 ]
```

Comment:
It's peculiar that we used to generate 6 files and now generate 2; the skycells got smaller rather than larger, so I expect to generate 4.  Do these two skycells files completely cover the L2 file?

Reply:
this appears to be working as expected; it's finding the unique skycells between the two matches: https://github.com/spacetelescope/romancal/blob/7592a904df89b99400718705c8746df0dadb9415/romancal/associations/mk_skycell_asn_from_skycell_list.py#L52-L54

Since the skycells are smaller, I would expect the same two footprints would have less overlap.

Reply:
this should be resolved by f4bfc3b2f5f844136f01c1ea336a994cb2360515 now